webodf/lib/gui/TrivialUndoManager.js

```diff
@@ -195,6 +204,51 @@ gui.TrivialUndoManager = function TrivialUndoManager(defaultRules) {
         eventNotifier.unsubscribe(signal, callback);
     };
 
+
+    /**
+     * @return {!number}
+     */
+    function currentUndoStateEditOpCount() {
+        /**
+         * @param {!number} count
+         * @param {!ops.Operation} op
+         * @return {!number}
+         */
+        function countEditOps(count, op) {
+            return count + (undoRules.isEditOperation(op) ? 1 : 0);
+        }
+        return currentUndoState.reduce(countEditOps, 0);
```

Comment:
This is potentially a very large collection, as a continuous state is things such as typing sentences + moving cursors. E.g., typing a paragraph could potentially be one continuous state.